### PR TITLE
vcredist: Add multiple portable version and update dependents

### PR DESCRIPTION
**Pull request by alexchapar**
## Description

Some apps requires a version of "Visual C++ Redistributable". Versions that were able to work as "portable" were added (based on the ones existing in the "extras bucket" with the addition in the PATH.

Apps that had dependcy over them has also been updated.

## Related Tickets & Documents

Not related

### Check Hashes?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Check Url?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Check the last version?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added a description in the manifest?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Beautified the json manifest format?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added some notes in the manifest?

- [x] 👍 yes  (for "vcredist")
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added/Needed any dependencies?

- [x] 👍 yes
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

## Added/Needed any pre-installation steps to perform?

- [x] 👍 yes (vcredist*: added for the "portable")
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

## Added/Needed any post-installation steps to perform?

- [x] 👍 yes (vcredist* : remove the old way to install)
- [ ] 🙋 yes, but I need help
- [ ] 🙅 no, because they aren't needed

## [optional] What gif/emoji best describes this PR or how it makes you feel?

:sweat:

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### dotnet-sdk

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

### mongodb

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

### php

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate
- [x] Autoupdate Hash Extraction

### vcredist

- [x] Description
- [x] License
- [x] Hashes

### vcredist2012

- [x] Description
- [x] License
- [x] Hashes

### vcredist2013

- [x] Description
- [x] License
- [x] Hashes

### vcredist2019

- [x] Description
- [x] License
- [x] Hashes

### windows-terminal

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate

### zoom

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate

**Review by Bibz87-CG — APPROVED** [review]

**Pull request merged**
